drivers/gpu/drm/starfive/starfive_drm_drv.h

```diff
@@ -14,6 +14,7 @@
 struct starfive_drm_private {
 	struct drm_fb_helper fbdev_helper;
 	struct drm_gem_object *fbdev_bo;
+	/* Protects mm. */
```

Comment:
This comment is not really helpful. What exactly is the "mm" that it is protecting? The drm_mm structure below?

Reply:
`mm` is the name of the variable this mutex should protect... But I just noticed that both `mm_lock` and `mm` are not used, so we should probably remove them.